locales/en-US.json

```
@@ -233,6 +224,32 @@
       "name": "Users"
     }
   },
+  "settings": {
+    "settings": "Settings",
+    "help": "Help",
+    "devices": {
+      "deleteAll": "Delete all",
+      "deleteDeviceSuccess": "Device deleted successfully",
+      "deleteDeviceError": "Error deleting device",
+      "deleteAllDevicesSuccess": "All devices deleted successfully",
+      "deleteAllDevicesError": "Error deleting all devices",
+      "noDevicesFound": "No devices found",
+      "delete": "Delete",
+      "appName": "App name",
+      "appVersion": "App version",
+      "lastActive": "Last active",
+      "userName": "Username",
+      "devices": "Devices"
+    },
+    "logsAndActivity": {
+      "activity": "Activity",
+      "failedGetActivity": "Unable to get activity",
+      "failedGetLogs": "Unable to get logs",
+      "logs": "Logs",
+      "noActivityFound": "No activities found",
+      "noLogsFounr": "No logs found"
```

```suggestion
      "noLogsFound": "No logs found"
```

Remember to update the key in the code as well